### PR TITLE
Fix test `02725_database_hdfs.sh` 

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit a5b9ba51cd0b0a0fb63172ddd5aa1fa6286e1771 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/56457/a5b9ba51cd0b0a0fb63172ddd5aa1fa6286e1771/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6797854179">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/56457/a5b9ba51cd0b0a0fb63172ddd5aa1fa6286e1771/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: tests/queries/0_stateless/02725_database_hdfs.sh [hunk 1]
```diff
@@ -9,25 +9,35 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
...
+    echo "Insert failed!"
+    exit 1
+fi
 
 #################
 echo "Test 1: select from hdfs database"
 
 # Database without specific host
 ${CLICKHOUSE_CLIENT} --multiline --multiquery -q """
 DROP DATABASE IF EXISTS test1;
```
Comment (davenger) [inline tests/queries/0_stateless/02725_database_hdfs.sh:29]:
```suggestion
DROP DATABASE IF EXISTS test_hdfs_1;
```

File: tests/queries/0_stateless/02725_database_hdfs.sh [hunk 2]
```diff
@@ -9,25 +9,35 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
...
-USE test1;
+CREATE DATABASE test_hdfs_1 ENGINE = HDFS;
+USE test_hdfs_1;
 SELECT * FROM \"hdfs://localhost:12222/test_02725_1.tsv\"
 """
 ${CLICKHOUSE_CLIENT} -q "SHOW DATABASES;" | grep test1
 
 # Database with host
 ${CLICKHOUSE_CLIENT} --multiline --multiquery -q """
 DROP DATABASE IF EXISTS test2;
```
Comment (davenger) [inline tests/queries/0_stateless/02725_database_hdfs.sh:38]:
```suggestion
DROP DATABASE IF EXISTS test_hdfs_2;
```

File: tests/queries/0_stateless/02725_database_hdfs.sh [hunk 3]
```diff
@@ -37,27 +47,27 @@ echo "Test 2: check exceptions"
 
 ${CLICKHOUSE_CLIENT} --multiline --multiquery -q """
 DROP DATABASE IF EXISTS test3;
```
Comment (davenger) [inline tests/queries/0_stateless/02725_database_hdfs.sh:49]:
```suggestion
DROP DATABASE IF EXISTS test_hdfs_3;
```

File: tests/queries/0_stateless/02725_database_hdfs.sh [hunk 4]
```diff
@@ -37,27 +47,27 @@ echo "Test 2: check exceptions"
...
-CREATE DATABASE test3 ENGINE = HDFS('abacaba');
+CREATE DATABASE test_hdfs_3 ENGINE = HDFS('abacaba');
 """ 2>&1 | tr '\n' ' ' | grep -oF "BAD_ARGUMENTS"
 
 ${CLICKHOUSE_CLIENT} --multiline --multiquery -q """
 DROP DATABASE IF EXISTS test4;
```
Comment (davenger) [inline tests/queries/0_stateless/02725_database_hdfs.sh:54]:
```suggestion
DROP DATABASE IF EXISTS test_hdfs_4;
```

File: tests/queries/0_stateless/02725_database_hdfs.sh (hunk 1 above)
Comment (kssenii) [reply tests/queries/0_stateless/02725_database_hdfs.sh:29]: thank you

**Review by davenger — APPROVED** [review]

**Comment by kssenii** [comment]
> Stateless tests (msan) [1/6] — fail: 1

02896_memory_accounting_for_user - not related to changes

> Stateless tests (ubsan) [2/2] — fail: 1

02475_bson_each_row_format - not related to changes

> Stateless tests flaky check (asan) — fail: 2, passed: 269 

I think this is unrelated to changes failure, these two failures happen in the test (`02113_hdfs_assert`) in which I changed only grepped error code (because I changed the thrown error code from code), and this failure could be explainable by too many concurrent queries to hdfs because of the flaky check nature.

**Pull request merged**